.code-samples.meilisearch.yaml

```diff
@@ -975,6 +929,161 @@ getting_started_search_md: |-
     }
   ```
   [About this SDK](https://www.github.com/meilisearch/meilisearch-swift)
+
+getting_started_update_rankingRules: |-
+  let rankingRules: [String] = [
+    "exactness",
+    "words",
+    "typo",
+    "proximity",
+    "attribute",
+    "sort",
+    "release_date:asc",
+    "rank:desc"
+  ]
+  client.index("movies").updateRankingRules(rankingRules) { (result: Result<Task, Swift.Error>) in
+    switch result {
+    case .success(let task):
+      print(task)
+    case .failure(let error):
+      print(error)
+    }
+  }
+getting_started_update_searchableAttributes: |-
+  let searchableAttributes: [String] = ["title"]
+  client.index("movies").updateSearchableAttributes(searchableAttributes) { (result: Result<Task, Swift.Error>) in
+      switch result {
+      case .success(let task):
+          print(task)
+      case .failure(let error):
+          print(error)
+      }
+  }
+getting_started_update_stop_words: |-
+  let stopWords: [String] = ["the"]
+  client.index("movies").updateStopWords(stopWords) { (result: Result<Task, Swift.Error>) in
+      switch result {
+      case .success(let task):
+          print(task)
+      case .failure(let error):
+          print(error)
+      }
+  }
+getting_started_check_task_status: |-
+  client.index("movies").getTask(taskUid: 0) { (result: Result<Task, Swift.Error>) in
+      switch result {
+      case .success(let task):
+          print(task)
+      case .failure(let error):
+          print(error)
+      }
+  }
+getting_started_synonyms: |-
+  let synonyms: [String: [String]] = [
+     "winnie": ["piglet"],
+     "piglet": ["winnie"]
+  ]
+   client.index("movies").updateSynonyms(synonyms) { (result: Result<Task, Swift.Error>) in
+       switch result {
+       case .success(let task):
+           print(task)
+       case .failure(let error):
+           print(error)
+       }
+   }
+getting_started_update_displayedAttributes: |-
+  let displayedAttributes: [String] = ["title", "overview", "poster"]
+  client.index("movies").updateDisplayedAttributes(displayedAttributes) { (result: Result<Task, Swift.Error>) in
+       switch result {
+       case .success(let task):
+           print(task)
+       case .failure(let error):
+           print(error)
+       }
+   }
+getting_started_communicating_with_a_protected_instance: |-
+  let client = MeiliSearch(host: "http://localhost:7700", apiKey: "apiKey")
+  client.index("movies").search { (result: Result<SearchResult<Movie>, Swift.Error>) in
+      switch result {
+      case .success(let searchResult):
+          print(searchResult)
+      case .failure(let error):
+          print(error)
+      }
+  }
+getting_started_add_meteorites: |-
+  let path = Bundle.main.url(forResource: "meteorites", withExtension: "json")
```

Comment:
FYI (curiosity): `Bundle.main` is only available in apps with the SPM, but not with cocoapods.

Reply:
I have updated the way to read from a file in all examples :)